### PR TITLE
fix(data retent): new release

### PR DESCRIPTION
**Pull request by zuluecho9**
NR-61417. Work done: 

* Clarify how some customers (but not all) can now self-serve data retention upgrade (including Data Plus) via the UI. 
* Assorted changes to readability/clarity
* Clarify how the rules work for upgrading retention. 
* Promote 'Data Plus' option a bit more prominently. 
* Add in mentions of related features (reducing data ingest, other data features)

### CODE REVIEW

File: src/content/docs/data-apis/manage-data/manage-data-retention.mdx [hunk 1]
```diff
@@ -16,38 +16,42 @@ import accountsEditRetentionSettings from 'images/accounts_screenshot-crop_edit-
 
 import accountsCopyDataRetention from 'images/accounts_screenshot-crop_copy-data-retention.png'
 
-With New Relic's **Data retention** UI page, you can review and manage how long different types of data are stored. Keep data for shorter periods to protect sensitive information and return queries faster, or keep it for longer periods to be able to see and analyze data and trends over a longer period. 
+On the **Data retention** UI page, located in the [data management hub]((https://one.newrelic.com/data-management-hub), you can review and manage how long different types of data are stored. You may want shorter retention for some data types in order to protect sensitive information, or you may want longer retention to be able to see and analyze data and trends over a longer period.
```
Comment (jeff-colucci) [inline src/content/docs/data-apis/manage-data/manage-data-retention.mdx:19]: Looks like a mis-formatted link.

**Review by jeff-colucci — APPROVED** [review]
Checked the diffs and the preview output, both look good. Only problem was the broken link, once that is fixed it should be good to go.

File: src/content/docs/data-apis/manage-data/manage-data-retention.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/data-apis/manage-data/manage-data-retention.mdx:19]: thanks!

File: src/content/docs/data-apis/manage-data/manage-data-retention.mdx [hunk 2]
```diff
@@ -676,43 +680,43 @@ Metric retention periods are not editable.
...
-* Once a higher retention period is set, reducing retention for specific data sources has no impact on billing.
+* If you haven't already, switch to [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), which offers significantly longer data retention and other benefits.
+* For Data Plus or original data option: add data retention. 30 days of additional data retention for your organization costs $0.05 per GB ingested per 30 days.
 
-### Other notes on reducing retention [#reduce-retention]
+Important points about increasing retention: 
 
-When you reduce the retention period for a specific data source, existing data of that type won't immediately disappear. That data might be visible and returned in queries for up to several weeks. We have safeguards in place to ensure you don't accidentally delete data, but this also means that reducing retention values isn't a good strategy for quickly [removing data](#data-deletion). 
+* Retention is increased across the board, for all data types listed in the **Data retention** UI. It cannot be increased only for specific data types.
```
Comment (jeff-colucci) [inline src/content/docs/data-apis/manage-data/manage-data-retention.mdx:694]: Should we add those data types here or is there another place where they are referenced?

**Review by jeff-colucci — DISMISSED** [review]
Only had a single small thought on one little piece but overall looks good to me. The change isn't worth holding up the merge so I am approving merging.

**Review by jeff-colucci — APPROVED** [review]
Looks good

**Pull request merged**